assets/oauth-openshift/oauth-server-deployment.yaml

```diff
@@ -69,6 +70,7 @@ spec:
 {{ if .MasterPriorityClass }}
       priorityClassName: {{ .MasterPriorityClass }}
 {{ end }}
+      terminationGracePeriodSeconds: 90
```

Comment:
In my testing these pods were OK but the OpenShift oauth API server pods that needed this increase.

Reply:
reverted